specification/components/r4/schemas/responses/UnsupportedMediaType.yaml

```diff
@@ -1,5 +1,20 @@
-description: Unsupported Media Type
+description: |
+  Where status code 415 (Unsupported Media Type) is returned then an OperationOutcome will be included in the body, as detailed below. 
```

Comment:
[NHSDigital-OperationOutcome](https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome) instead of OperationOutcome